### ISSUE TITLE
Is it possible to block specific domains?

### ISSUE THREAD

**Issue by vachaganbalayan** [Jigsaw-Code/outline-server#676]
I'd like to block specific domains entirely from my self hosted outline vpn.

Is it possible to block things like facebook youtube so that those are completely inaccessible while you're using the vpn...

**Comment by williamgh2019** [comment]
Yes, I would love to have this feature and also please consider allow to block porn & ads list.

**Comment by MHitMaN** [comment]
+1

**Comment by beingke** [comment]
+2

**Comment by lysukhin** [comment]
@fortuna Hello, are there any plans on this?

**Comment by nimaadib** [comment]
+1

**Comment by powerdot** [comment]
+1

**Comment by VPNCI** [comment]
Look at dnsmasq.

**Comment by taheri79** [comment]
+1

**Comment by olivehair** [comment]
+1

**Comment by piskov** [comment]
> You can use ‹…› `/etc/hosts.deny`

@murka could you please elaborate? I have an outline server on ubuntu and if i specify any item in either hosts or hosts.deny on the server, none are actually blocked on a client that is connected via that outline server.

**Comment by liqlos** [comment]
+

**Comment by evtaranov** [comment]
I'm still really looking forward to this feature 😢

**Comment by navinweb** [comment]
has anyone figured out how to do that?

**Comment by rafaellaurindo** [comment]
+1
